xbmc/music/tags/MusicInfoTag.cpp

```diff
@@ -737,6 +772,7 @@ void CMusicInfoTag::Serialize(CVariant& value) const
   value["albumid"] = m_iAlbumId;
   value["compilationartist"] = m_bCompilation;
   value["compilation"] = m_bCompilation;
+  value["boxset"] = m_bBoxset;
```

Comment:
`m_iDiscTotal` also needs serialzation

I am in two minds. Serialization is for JSON, and it makes sense some to leave changes to it to when the API schema addition is implemented. Adding it now does no harm, but obviously needs to be consistent - add all fields or don't add any.  But like with CSong.Serialize should it have a "todo" so that the change can be tied to the JSON change?

So add all fields and inline comment, or don't change the Serialize methods yet, up to you.